fusequery/query/src/sql/plan_parser.rs

```diff
@@ -563,18 +563,18 @@ impl PlanParser {
     }
 
     fn create_relation(&self, relation: &sqlparser::ast::TableFactor) -> Result<PlanNode> {
-        use sqlparser::ast::TableFactor::*;
+        use sqlparser::ast::TableFactor as Ast;
 
         match relation {
-            Table { name, args, .. } => {
+            Ast::Table { name, args, .. } => {
```

Comment:
Why not just `TableFactor::Table`?

Reply:
yeah, `TableFactor::Table` will be better

Reply:
PTAL